src/mflux/models/qwen/model/qwen_text_encoder/qwen_encoder.py

```
@@ -29,13 +894,448 @@ def __init__(
+                image_embeds_split = self.get_image_features(pixel_values, image_grid_thw)
+                # Concatenate embeddings in order: [image1_embeds, image2_embeds, ...]
+                # This order must match the order of images in the prompt
+                image_embeds = mx.concatenate(image_embeds_split, axis=0)
+
+                print(f"🔎 QwenEncoder: Vision tower output: {image_embeds.shape}")
+                print(f"🔎 QwenEncoder: Number of image splits: {len(image_embeds_split)}")
+                if len(image_embeds_split) > 1:
+                    print(f"🔎 QwenEncoder: Split sizes: {[e.shape[0] for e in image_embeds_split]}")
+            else:
+                print(f"🔎 QwenEncoder: Unknown pixel_values format with shape {pixel_values.shape}")
+                raise ValueError(f"Unsupported pixel_values format: {pixel_values.shape}")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)